knowledge-base/style-radentry-focused-state-telerik-maui.md

```diff
@@ -0,0 +1,63 @@
+type: how-to
+page_title: How to Style RadEntry with Focused State in Telerik MAUI Version 8.0.0+
+slug: style-radentry-focused-state-telerik-maui
+tags: radentry, styling, visual states, telerik, maui, upgrade, focused state
+res_type: kb
+---
+
+## Environment
+
+| Versions | Product | Author | 
+| --- | --- | ---- | 
+| 7.1.0 vs. latest | Telerik UI for .NET MAUI RadEntry | [Dobrinka Yordanova](https://www.telerik.com/blogs/author/dobrinka-yordanova) | 
+
+## Description
+
+After updating my application from Telerik MAUI 7.1.0 to 8.0.0 or the latest version, the styles set on `RadEntry` controls, such as colored borders and backgrounds, disappear when the controls are focused. This knowledge base article also answers the following questions:
+- How can I apply border color and thickness to `RadEntry` in Telerik MAUI after an update?
+- How do I maintain `RadEntry` background color when it's focused in the latest Telerik MAUI version?
+- What changes are necessary for styling `RadEntry` upon focusing after migrating to Telerik MAUI 8.0.0 or newer?
+
+## Solution
+
+To style the `RadEntry` control, including maintaining or altering the border color, thickness, and background color when the control is focused, you need to apply specific styles to the `Focused` visual state of the control. This approach is necessary due to changes in the `RadEntry` API and the introduction of a new theming mechanism in Telerik MAUI version 8.0.0 and later.
+
+### Set the Focused Visual State
+
+First, familiarize yourself with the [RadEntry Styling](https://docs.telerik.com/devtools/maui/controls/entry/styling) documentation and the [Visual States](https://docs.telerik.com/devtools/maui/controls/entry/visual-states) guide. These resources provide essential information on how to style the `RadEntry` control effectively.
+
+### API Changes from Telerik MAUI 7.1.0 to 8.0.0 and Later
+
+The `RadEntry` control underwent significant changes in the transition from version 7.1.0 to 8.0.0.
+
+ **`RadEntry` Inheritance**: Prior to version 8.0.0, `RadEntry` inherited directly from Microsoft MAUI Entry. In version 8.0.0 and later, `RadEntry` is a composite control using the [`RadTextInput`]({%slug entry-textinput%}) control internally for text input.
+
+The table below summarizes the key changes relevant to styling:
+
+| Telerik.UI.for.MAUI <= 7.1.0 | Telerik.UI.for.MAUI >= 8.0.0 |
+| ------------ | ----------- |
+| `ClearButtonColor` | Set the `TextColor` in the `ClearButtonStyle` (`Style` with target type `RadTemplatedButton`). |
+| `HoveredClearButtonColor` | Use `PointerOver` `VisualState` in the `ClearButtonStyle` (`Style` with target type `RadTemplatedButton`) and set `TextColor`. |
+| `PressedClearButtonColor` | Use `Pressed` `VisualState` in the `ClearButtonStyle` (`Style` with target type `RadTemplatedButton`) and set `TextColor`. |
+| - | [`VisualStates`]({%slug entry-visual-states%}). |
```

Comment:
Use NA instead of "-" wherever the feature is missing in the previous versions. Please apply this change everywhere in the table.
```suggestion
| NA | [`VisualStates`]({%slug entry-visual-states%}). |
```